typescript-functions/main.ts

```diff
@@ -0,0 +1,47 @@
+function convertMinutesToSeconds(minutes: number): number {
+  return minutes * 60;
+}
+
+console.log('total seconds in 5 minutes: ', convertMinutesToSeconds(5));
+console.log('total seconds in 10 minutes: ', convertMinutesToSeconds(10));
+
+function greet(name: string): string {
+  return `Hey ${name}`;
+}
+console.log(greet('Beavis'));
+console.log(greet('SpongeBob'));
+
+const getArea: (width: number, height: number) => number = (width, height) => {
+  return width * height;
+};
```

Comment:
Works, but we can simplify this. Right now, you are currently typing the whole arrow function twice, but we only need to worry about the parameters and the `return` value. You can do something like this instead

```
const getArea = (width: number, height: number): number => {
  const area: number = width * height;
  return area;
};
```